### ISSUE TITLE
Missing 'draw_detected_object_boxes' process

### ISSUE THREAD

**Issue by emmerg** [Kitware/kwiver#635]
Building from the master branch, commit v1.2.0-301-gb21f695, on Ubuntu 16.04, kernel 4.4.0-137-generic

I build fletch as outlined in the fletch repo, specifically with `-Dfletch_ENABLE_ALL_PACKAGES=ON`. After the `make` and `make install`, I pulled Kwiver and made it again following instructions outline in the repo's README. In order to test my installation, I tried to run the examples in the [tutorials](https://kwiver.readthedocs.io/en/latest/tutorials.html) linked in the readme.

The numbers pipeline (from the repo's README), and the Hough Detection examples work fine. However, when I try to run the example detector, I see the following:

```
ubuntu@kwiver-machine:~/kwiver/build/release/bin$ ./pipeline_runner -p ../examples/pipelines/example_detector_on_image.pipe
2018-10-25 09:07:19.696 INFO sprokit.modules.modules(68): Loading python modules
2018-10-25 09:07:19.994 INFO sprokit.modules.modules(68): Loading python modules
Exception caught: There is no such process of type 'draw_detected_object_boxes' in the registry, thrown from /home/ubuntu/kwiver/src/sprokit/src/sprokit/pipeline/process_factory.cxx:127
```

Has this example become deprecated in recent builds? Or does this indicate some failure in my build process?

**Comment by linus-sherrill** [comment]
That process has been deprecated. I will update the example to use the new drawing process.

**Comment by Cookt2** [comment]
Fixed in #638. Closing.
